modules/ossm-about-istio-high-availability.adoc

```diff
@@ -0,0 +1,14 @@
+// This module is used in the following assemblies:
+// * service-mesh-docs-main/install/ossm-installing-service-mesh.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-istio-high-availability_{context}"]
+= About Istio High Availability
+
+Running the {istio} control plane in High Availability (HA) mode prevents single points of failure, and ensures continuous mesh operation even if an `istiod` pod fails. By using HA, if one `istiod` pod becomes unavailable, another one continues to manage and configure the {istio} data plane, preventing service outages or disruptions. HA provides scalability by distributing the control plane workload, enables graceful upgrades, supports disaster recovery operations, and protects against zone-wide mesh outages.
```

Comment:
"By using HA, if one `istiod` pod becomes unavailable, another one continues..."
@rh-tokeefe does this make sense or it should be rephrased?